docs/Tutorial_DiscordMERLIN.ipynb

```diff
@@ -0,0 +1,993 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #59.            eligible_cands[k_discord_idx[i]-excl_zone:k_discord_idx[i]+excl_zone+1] = False
</pre><p>If you merge the latest commit from <code>main</code>, you can replace this line with:</p><p> </p><pre class="ql-syntax" spellcheck="false">core.apply_exclusion_zone(eligible_cands, k_discord_idx[i], excl_zone, val=False)
</pre><p> </p><p>This will help ensure that exclusion zone definitions are kept consistent across the code base and developers don't have to think about the logic of whether the start/end are inclusive or exclusive.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/TDAmeritrade/stumpy/pull/505/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='40'/>

Reply:
This is very good! Will be fixed in next commits